buildscripts/cstor-csi-driver/entrypoint.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Why do we need to change the shell?

Reply:
We can remove this file as it is not used in any dockerfile.

Reply:
not required